src/app/application.cpp

```diff
@@ -278,7 +279,7 @@ void Application::torrentFinished(BitTorrent::TorrentHandle *const torrent)
         program.replace("%T", torrent->currentTracker());
         program.replace("%I", torrent->hash());
 
-        QProcess::startDetached(program);
+        std::thread ([program]() { return system(program.toStdString().c_str()); }).detach();
```

Comment:
>But we support the limited set of platforms. And thus you can easily write such function.
Update: it is very simple: `CreateProcess("cmd.exe",...)` on windows and `posix_spawn()` or `vfork()` + `execl()` "/bin/sh" everywhere else.

@Chocobo1 You can see `QProcessPrivate::startDetached` for example.